packages/Uploader/src/types.js

```diff
@@ -0,0 +1,8 @@
+export default {
+  IDLE: "idle",
+  PROCESSING: "processing",
+  WAITINGFORSERVER: "waiting-for-server",
+  SUCCESS: "success",
+  ERROR: "error",
+  CANCEL: "cancel",
+};
```

Comment:
Seems pretty specific to the `<Uploader>`, would this actually be a candidate to migrate to a `types` package?

Reply:
I think this is too specific where status and props are more generic, it might not make sense to move it outside of here.